app/boards/shields/reviung34/Kconfig.defconfig

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 The ZMK Contributors
+# SPDX-License-Identifier: MIT
+
+if SHIELD_REVIUNG34
+
+config ZMK_KEYBOARD_NAME
+    default "Reviung34"
```

```suggestion
    default "REVIUNG34"
```

Creator seems to call it REVIUNG34: https://github.com/gtips/reviung